lib/openxml/drawingml/properties/custom_geometry.rb

```diff
@@ -0,0 +1,18 @@
+module OpenXml
+  module DrawingML
+    module Properties
+      class CustomGeometry < SimplePropertyContainerProperty
+        namespace :a
+        tag :custGeom
+
+        property :shape_adjustment_list
+        property :geometry_guide_list
+        # property :adjustable_handle_list # ahLst
+        # property :connection_list # cxnLst
+        # property :rectangle # rect
+        property :path_list # Required!
```

Comment:
What's with commented-out properties?

The `# Required!` comment — again, did you implement that?